source/sass/blocks/company-item.scss

```
@@ -0,0 +1,41 @@
+//блок волонтерство
+.company-item__heading .index-block__title {
+    font-size: 38px;
+    font-weight: 900;
+    line-height: 48px;
+    color: #000000;
+}
+
+.company-item__text {
+    font-family: 'CeraKFC-400';
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 20px;
+    letter-spacing: 0em;
+    color: #000000;
+}
+
+//не применяется, не могу перебить общие стили от h4
+.company-item__text .h4 {
```

Так там же тег `<h4>`, а не класс
```suggestion
//отлично применяется =)
.company-item__text h4 {
```